modules/geo-layers/src/tile-3d-layer/tile-3d-layer.js

```diff
@@ -167,11 +167,12 @@ export default class Tile3DLayer extends CompositeLayer {
     if (!timeline || !viewportsNumber || !tileset3d) {
       return;
     }
-    const frameNumber = tileset3d.update(Object.values(viewports));
-    const tilesetChanged = this.state.frameNumber !== frameNumber;
-    if (tilesetChanged) {
-      this.setState({frameNumber});
-    }
+    tileset3d.selectTiles(Object.values(viewports)).then(frameNumber => {
```

Comment:
can we use async / wait syntax instead of old style `.then`?
```suggestion
    const frameNumber = await tileset3d.selectTiles(Object.values(viewports));
```